py/greetings.py

```diff
@@ -1,19 +1,36 @@
+import html
 import dominate
 from dominate.tags import *
 
-def _greetings():
-    """Returns the generated HTML greetings. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+def _greetings(week):
+    """This function generates a greetings section below the header using table elements and returns it as HTML. 
+    The greetings consists of two subheaders separated by a horizontal line. The first subheader prints the function argument,
+    which will most likely be "HAPPY WEEK # EVERYONE" and the second subheader prints, "Here are some things to get excited about!".
+    The horizontal line in the middle is a bordered yellow div.
 
     Args:
-        TODO
+        week: A string containing the current week number.
 
     Returns:
-        TODO
+        An HTML document object containing the newsletter's greetings. The greetings consists of two table elements within 
+        a single table row. Each of the two rows contains the messages described above and the rows are separated by the horizontal
+        line div.
 
     Raises:
-        TODO
+        None
     """
 
-    return h2('This is the greetings.')
+    with div() as greeting:
+        # Greeting
+        with tr().add(td(cls='td')).add(table(cls='table')).add(tr()).add(td(cls='td greeting')):
+            # First subheader
+            with table(cls='table').add(tr()).add(td(cls='td')):
+                # Changes according to argument week
+                h2(week)
```

Comment:
So now we are having the user input the entire "Happy Week 7 Everyone" string? If so, can we add the greetings in the second subheader as a parameter?

Reply:
I did this since that's what I thought you meant from an earlier requested change. But just inputting the week # might be easier in the use case.

Reply:
Added both parameters